components/manifest-editor/src/components/manifest-platform-form.ts

```diff
@@ -1030,6 +1040,30 @@ export class ManifestPlatformForm extends LitElement {
                     
               </div>
           </div>
+          <div class="form-row">
+          <div class="form-field">
+            <div class="field-header">
+              <h3>Edge Side Panel</h3>
+              <a
+                href="https://github.com/MicrosoftEdge/MSEdgeExplainers/blob/main/SidePanel/explainer.md"
+                target="_blank"
+                rel="noopener"
+              >
+                <img src="/assets/tooltip.svg" alt="info circle tooltip" />
+                <p class="toolTip">
+                  Click for more info on the Edge Side Panel option in your manifest.
+                </p>
+              </a>
+            </div>
+            <p>Indicates whether your PWA supports the side panel in the Edge Browser</p>
```

Comment:
```suggestion
            <p>Indicates whether your PWA supports the side panel in Microsoft Edge</p>
```

nit: usually Edge browser is mentioned as Microsoft Edge for branding purposes.